multimodal-search-tirg/pods/query-image-read.yml

```diff
@@ -0,0 +1,21 @@
+!ImageReader
```

Comment:
Why do not we merge this with the `index reader`? since they will have different drivers for different kind of Requests they should be able to be described in the same file